crates/syn-solidity/src/lib.rs

```diff
@@ -84,7 +84,7 @@ pub mod visit_mut;
 pub use visit_mut::VisitMut;
 
 mod yul;
-pub use yul::YulBlock;
+pub use yul::*;
```

Comment:
I know it's a chore, but can you list all items instead of glob? On VS Code I use `loop { <Comma>; Ctrl + I; Enter }` to write all the items that can be exported until you reach `self`